gists/frontend-pull-request-guidelines.md

```diff
@@ -1,6 +1,23 @@
 # Frontend Pull Request Guidelines
 This document details guidelines for reviewing frontend (especially LPL React-flavored) code. Some of these suggestions may apply more broadly to software development in general, but are mentioned here specifically based on the frequency with which they have been observed in client-side code.
 
+## General
+- Strive for consistency. Review other areas of the code for patterns, copy and paste, make things easier for yourself.
+  - However, don't do it blindly. If you know of a better way, bring it up to the team. Maybe we should change the pattern holistically.
```

Comment:
Speaking of consistency 😅 , can we choose to either end all with punctuation or none? I believe the current established convention is none